packages/cadl-typescript/src/transform/transformAnnotationDetails.ts

```diff
@@ -24,9 +30,23 @@ export function transformAnnotationDetails(
   }
   // TODO: Remove this when @pageable is finally removed.
   const nextLinks = new Set<string>();
-  const [services, _diagnostics] = getAllHttpServices(program);
-  const routes = services.flatMap((service) => service.operations);
-  for (const route of routes) {
+  const operationGroups = listOperationGroups(program, client);
```

Comment:
Does that mean the `@operationGroup` work now?

Reply:
not really, as @operationGroup are still in discussion, there maybe some changes, Don't think we should rush to support it right now.